### ISSUE TITLE
[DUPLICATE]Unable apply changes to external monitor scale.

### ISSUE THREAD

**Issue by isantop** [pop-os/pop#6]
_From @jgreat on October 23, 2017 18:0_

If this is a bug, please use the template below. If this is a feature request, question, or a general discussion topic - please post on our subreddit https://reddit.com/r/pop_os - as that is the proper forum for those types of posts.

**Reporter info**

```
* Distribution - (run ```cat /etc/os-release```)
NAME="Pop!_OS"
VERSION="17.10 (Artful Aardvark)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Pop!_OS 17.10 (Artful Aardvark)"
VERSION_ID="17.10"
HOME_URL="https://system76.com/pop"
SUPPORT_URL="http://support.system76.com/"
BUG_REPORT_URL="https://github.com/system76/pop-distro/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=artful
UBUNTU_CODENAME=artful

 
 * Related Application and/or Package Version - Not sure, what is the setting package 
```

**Issue/Bug Description**

Just changing the scale on external monitor doesn't show "Apply" button 

**Steps to reproduce (if you know)**

Open up settings -> Devices -> Displays
Select external monitor
Only change the scale (100 to 200)
Screen doesn't scale and no Apply button shows up.

![image](https://user-images.githubusercontent.com/5324639/31904905-1814009a-b7f2-11e7-9f74-9bcb857056e0.png)

**Expected behaviour**

Screen scales or apply button is shown.

**Other Notes**

_Copied from original issue: pop-os/iso#185_

**Comment by isantop** [comment]
_From @jeffreyscottgraham on October 24, 2017 15:45_

Must be a bug.
Related: no scale factor for (some?) external monitors

**Comment by isantop** [comment]
@jeffreyscottgraham That's intended behavior. The Display Settings are only supposed to list scales that make sense for a given physical size and resolution. If 2x would be too large for that display, they don't show it.

**Comment by isantop** [comment]
_From @dgoerlich on October 25, 2017 21:26_

I'm also having an issue with a lack of an Apply button.  What I found was that I was able to get it to show by changing the "Display Mode" from "Join Displays" to "Mirror" and back again, and then had the Apply button which allowed me to save Display Arrangement and resolution changes.

Unfortunately the "Display Mode" doesn't make sense to show when I have two external monitors attached (HDMI and MiniDP), so I'm unable to save any changes as soon as I add my second external monitor.
